plugin/vcs/gitlab/gitlab.go

```diff
@@ -242,7 +242,7 @@ func (o oauthResponse) toVCSOAuthToken() *vcs.OAuthToken {
 	return oauthToken
 }
 
-// ExchangeOAuthToken exchange oauth content with the provided authentication code.
+// ExchangeOAuthToken exchanges OAuth content with the provided authorization code.
```

Comment:
Docstrings in Go use third-person point of view; authorization is not the same thing as authentication.